test/documentation/seqan3.css

```diff
@@ -56,6 +56,7 @@ div.contents {
 
 #projectlogo {
     padding: 10px;
+    width: 150px;
```

Comment:
I added a small fix that makes the logo align to the right again (see https://docs.seqan.de/seqan/3.0.0/)